receiver/purefareceiver/config.go

```diff
@@ -61,6 +63,23 @@ type ReloadIntervals struct {
 }
 
 func (c *Config) Validate() error {
-	// TODO(dgoscn): perform config validation
-	return nil
+	var err error
+
+	if len(c.Settings.ReloadIntervals.Array.String()) == 0 {
+		err = multierr.Append(err, errors.New("Arrays not provided and is required"))
+	}
+	if len(c.Settings.ReloadIntervals.Host.String()) == 0 {
+		err = multierr.Append(err, errors.New("Hosts not provided and is required"))
+	}
+	if len(c.Settings.ReloadIntervals.Directories.String()) == 0 {
+		err = multierr.Append(err, errors.New("Directories not provided and is required"))
+	}
+	if len(c.Settings.ReloadIntervals.Pods.String()) == 0 {
+		err = multierr.Append(err, errors.New("Pods not provided and is required"))
+	}
+	if len(c.Settings.ReloadIntervals.Volumes.String()) == 0 {
+		err = multierr.Append(err, errors.New("Volumes not provided and is required"))
+	}
```

Comment:
Personal taste, is to compare with empty string for strings, and with len 0 for slice/map.

Reply:
@bogdandrutu thanks for this review. I agree with this

Reply:
This is similar to the Validate part of the purefb receiver. Once it's aligned with that, I'll merge this.